### ISSUE TITLE
`npm audit fix --force` reverts parcel@2.1.1 to 1.12.4

### ISSUE THREAD

**Issue by henriquemattos** [parcel-bundler/parcel#7550]
# 🐛 bug report

As of this moment, parcel@2.1.1 is the latest release of v2, but installing it via `npm install --save-dev parcel@2.1.1` generates an auditing report of 55 vulnerabilities (42 low, 13 moderate), and executing `npm audit fix --force` reverts parcel to `parcel@1.12.4`.

```sh
parcel % npm install parcel@2.1.1
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

added 668 packages, and audited 669 packages in 27s

141 packages are looking for funding
  run `npm fund` for details

55 vulnerabilities (42 low, 13 moderate)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

and fixing the audited packages:

```sh
parcel % npm audit fix --force
npm WARN using --force Recommended protections disabled.
npm WARN audit Updating parcel to 1.12.4,which is a SemVer major change.

```

## 🎛 Configuration (package.json)

`npm init -y && npm install --save-dev parcel@2.1.1`

```js

{
  "name": "parcel",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "parcel": "^1.12.4"
  }
}

```

## 🤔 Expected Behavior

I would expect a patched version 2 to be installed, not reverted to v1.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.1.1
| Node             | 17.3.0
| npm         | 8.3.0
| Operating System | macOS Monterey 12.1

**Comment by henriquemattos** [comment]
Also this is a problem because of issue #5943 (https://github.com/parcel-bundler/parcel/issues/5943)

**Comment by devongovett** [comment]
Security issues should be fixed in v2.2.1. Not sure what was up with npm audit fix, but seems like a bug in npm?
